False: Non-zero return code, probably a syntax error occurred\ntime=1637690660.7917757 gym-duckietown 2019.0.0

time=1637690661.841688 Registering gym environment id: Duckietown-4way-v0
time=1637690661.8418536 Registering gym environment id: Duckietown-loop_dyn_duckiebots-v0
time=1637690661.8419511 Registering gym environment id: Duckietown-loop_empty-v0
time=1637690661.8421106 Registering gym environment id: Duckietown-loop_obstacles-v0
time=1637690661.842188 Registering gym environment id: Duckietown-loop_pedestrians-v0
time=1637690661.8422573 Registering gym environment id: Duckietown-small_loop-v0
time=1637690661.8423283 Registering gym environment id: Duckietown-small_loop_cw-v0
time=1637690661.8424003 Registering gym environment id: Duckietown-straight_road-v0
time=1637690661.8424716 Registering gym environment id: Duckietown-straight_road_10_tiles_no_start_tile-v0
time=1637690661.8425398 Registering gym environment id: Duckietown-straight_road_no_start_tile-v0
time=1637690661.8426366 Registering gym environment id: Duckietown-straight_road_trafficlight-v0
time=1637690661.8427598 Registering gym environment id: Duckietown-straight_road_with_side_floor-v0
time=1637690661.8428426 Registering gym environment id: Duckietown-udem1-v0
time=1637690661.842925 Registering gym environment id: Duckietown-zigzag_dists-v0
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/appuser/aido-auto/gym-duckietown/gym_duckietown/tasks/dont_crush_duckie/checker.py", line 6, in <module>
    checkers=[StopAtCoordinatesChecker()]
  File "/home/appuser/aido-auto/gym-duckietown/gym_duckietown/tasks/solution_evaluator.py", line 31, in __init__
    solution_file_as_module = importlib.import_module(self.args.solution_file_as_module)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/appuser/aido-auto/gym-duckietown/gym_duckietown/tasks/dont_crush_duckie/solutions/solution.py", line 28, in <module>
    domain_rand = False,
  File "/home/appuser/aido-auto/gym-duckietown/gym_duckietown/envs/duckietown_env.py", line 24, in __init__
    Simulator.__init__(self, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'dynamics_rand'
